### ISSUE TITLE
SPServiceAppSecurity: ExpandAccessLevel not recognized

### ISSUE THREAD

**Issue by paololec** [PowerShell/SharePointDsc#984]
#### Details of the scenario you tried and the problem that is occurring
Bug on SPServiceAppSecurity
#### Verbose logs showing the problem
PowerShell DSC resource MSFT_SPServiceAppSecurity  failed to execute Test-TargetResource functionality with error message: The term 
'ExpandAccessLevel' is not recognized as the name of a cmdlet, function, script file, or operable program.

#### Suggested solution to the issue
Add ExpandAccessLevel to function ???
Export-ModuleMember -Function *-TargetResource, ExpandAccessLevel

#### The DSC configuration that is used to reproduce the issue (as detailed as possible)
```powershell
$MembersToInclude = @();
             $MembersToInclude += MSFT_SPServiceAppSecurityEntry {
                Username    = $Credssp_searchcontent.UserName;
                EntryNumber = 1;
                AccessLevels = @("Full Access to Term Store");
            };
            SPServiceAppSecurity ServiceAppSecurityPermissionsManagedMetadata
            {
                SecurityType = "SharingPermissions";
                ServiceAppName = "Managed Metadata Service";
                PsDscRunAsCredential = $Credssp_admin;
                MembersToInclude = @($MembersToInclude);
            }
```

#### The operating system the target node is running
Win Server 2016
#### Version of SharePoint that is used (e.g. SharePoint 2016)
SharePoint 2019
#### Version and build of PowerShell the target node is running
 5.1.14393.2608       

#### Version of the DSC module that was used ('dev' if using current dev branch)
3.0

**Comment by scoreman** [comment]
Can only confirm that I'm experiencing the same issue as @paololec on SP2019 with DSC 3.0

**Comment by ykuijs** [comment]
Just merged PR #983 which fixed this issue. This will be included in v3.1 or you can grab the current bits from the Dev branch.

**Comment by ykuijs** [comment]
Thanks for reporting!
